marketplace_builder/pages/getting-started/users/creating-sign-up-for.liquid

```diff
@@ -0,0 +1,245 @@
+---
+converter: markdown
+metadata:
+  title: Creating a Sign Up Form
+slug: get-started/users/creating-sign-up-form
+---
+
+This guide will help you create sign up forms for users. 
+
+## Requirements
+
+To follow the steps in this tutorial, you should be familiar with the required directory structure for your code base, and understand the concept of users. You'll use the User Profiles created in a previous tutorial. 
+
+* [Required Directory Structure](/how-platformos-works/platformos-components)
+* [Users](/get-started/users/users)
+* [Creating a User Profile](/get-started/users/creating-user-profile) (previous tutorial)
+
+## Steps
+
+Creating a sign-up form is a two-step process:
+
+1. Create form file
+2. Configure form 
+
+### Step 1: Create form files 
+
+Create a form to sign up developers first. Create a liquid file in the `marketplace_builder/form_configurations/developer/` directory, and name it `developer_sign_up.liquid`. 
+
+Then create a liquid file in the `form_configurations/client/client_sign_up.liquid` directory named `client_sign_up.liquid`.  
+
+### Step 2: Configure forms 
+
+#### Developer form 
+
+Add the following content to the `developer_sign_up.liquid` sign up form:
+
+{% raw %}
+
+```liquid
+---
+name: developer_sign_up
```

Comment:
@pavelloz How about creating real files for such examples in `marketplace_builder` directory and then displaying fetching them with some graphql query? With that we could even write tests.

Reply:
hmm, now that we have cms-* endpoints in graphql query its almost possible - do we have option to get raw content of a file from gql (with yml and all that)?